src/gatsby/src/components/advisories/advisoryLegend.js

```diff
@@ -47,10 +47,15 @@ const AdvisoryLegend = () => {
         })}
       </div>
       <div className="mb-4 mt-3 px-md-5 mx-md-5 font-italic text-center">
-        For most up to date emergencies please visit BC Flood &amp; Wildfire
-        Service and learn how to be prepared for an emergency before it happens.
-        Updated Monday to Friday from 8:30 am to 4:30 pm, excluding statutory
-        holidays.
+        Advisories are updated Monday to Friday 8:30 to 4:30, excluding statutory holidays.<br />
+        Get up-to-date provincial emergency information for wildfires, floods, and highways from:<br />
+        <a href="https://www2.gov.bc.ca/gov/content/safety/wildfire-status">
+          BC Wildfire Service
+        </a>{", "}
```

Comment:
Just confirming, the Jira ticket shows bullets and not commas?

Reply:
Confirmed with Jane and Brett. We decided to change it to normal text with commas. 
Details: https://bcparksdigital.atlassian.net/browse/CM-633